app/core/constants.js

```diff
@@ -90,6 +90,16 @@ export const ENDED_ICO_TOKENS = [
   'DBC', 'RPX', 'RHT', 'QLC', 'NRVE', 'IAM', 'ONT', 'THOR', 'CGE', 'AVA', 'SWH', 'SWTH', 'EFX', 'MCT', 'PKC'
 ]
 
+export const DEPRECATED_TOKENS = [
+  '4b4f63919b9ecfd2483f0c72ff46ed31b5bbb7a4', // Phantasma SOUL
+  'a0b328c01eac8b12b0f8a4fe93645d18fb3f1f0a', // NKN
+  '7ac4a2bb052a047506f2f2d3d1528b89cc38e8d4', // quarteria
+  '23501e5fef0f67ec476406c556e91992323a0357', // Orbis
+  '442e7964f6486005235e87e082f56cd52aa663b8', // Ontology
+  '2e25d2127e0240c6deaf35394702feb236d4d7fc', // Narrative
+  '6d36b38af912ca107f55a5daedc650054f7e4f75' // Aphelion
```

Comment:
How do you suggest I go about verifying these are the correct deprecated script hashes?

Reply:
I looked at duplicates mainnet contracts where the oldest 1 isnt on the list here

But yeah, probably better to contact each owner to ask if they want to list it as deprecated

I can edit the PR to leave only our contract now (SOUL). Also NRV and Orbis i know have been deprecated